### PR TITLE
feat: add fixed property

### PR DESCRIPTION
**Pull request by gusnaughton**
# Alaska Airlines Pull Request

For placements with inconsistent root font-size values, add a fixed property that uses raw pixel values instead of rems.

## Type of change:

Please delete options that are not relevant.

- [x] New capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -14,6 +14,46 @@
...
+      font-size: 16px;
+      @include auro_breakpoint--sm {
+        font-size: 18px;
+      }
+    }
+  }
+  .dayOfTheWeek {
+    font-size: 18px;
+    @include auro_breakpoint--sm {
+      font-size: 18px;
```
Comment (geoffrich) [inline src/style.scss:34]: Should be 20px

File: src/style.scss [hunk 2]
```diff
@@ -14,6 +14,46 @@
...
+  .price {
+    font-size: 14px;
+    @include auro_breakpoint--sm {
+      font-size: 18px;
+    }
+  }
+  .price--long {
+    font-size: 12px;
+    @include auro_breakpoint--sm {
+      font-size: 16px;
```
Comment (geoffrich) [inline src/style.scss:52]: Should be 18px

File: src/style.scss [hunk 3]
```diff
@@ -14,6 +14,46 @@
   display: inline-block;
 }
 
+:host([fixed]) {
```
Comment (geoffrich) [inline src/style.scss:17]: There's a bunch of other properties set to rems as well (e.g. height, width, row size, padding). If we want the pane to display correctly we need to set those as well.

**Review by geoffrich — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 2 above)
Comment (geoffrich) [reply src/style.scss]: No longer needed since fixed styles are automatically generated.

File: src/style.scss (hunk 3 above)
Comment (geoffrich) [reply src/style.scss]: Fixed styles are now automatically generated.

**Review by geoffrich — APPROVED** [review]

**Comment by blackfalcon** [comment]
I really like this solution, but I am wondering if this needs to be more dynamic. The additional CSS is large and in-light of the conversations I am having with @geoffrich about bundle size, I am wondering if this should actually be a functional attribute that decides which CSS to load? 

I do see that the fixed-value CSS is smaller than the standard, so this is only addressing non-REM value CSS. But if we remove that plug-in then we could have a full stylesheet that is all the CSS with px values. 

The good news is that the API doesn't change. The return is simply refactored.

**Comment by geoffrich** [comment]
I'd be in favor of finding a way to conditionally load this. Not entirely sure what that would look like though. We could subclass it like we do for light-dom (auro-pane-fixed?) but I'm not in love with that, especially since it would get messy if applied to other components with a light version (auro-button-light, auro-button-fixed, auro-button-light-fixed...)

Would dynamic imports be something we could use here?

**Comment by blackfalcon** [comment]
I don't have all the details thought out ... I am pondering ideas and will come back to this.

**Pull request merged**